tests/unit/records/schema/test_pandas.py

```diff
@@ -83,7 +83,7 @@ def test_pandas_numeric_types_and_constraints(self):
             ('float32', np.float32),
             ('float64', np.float64),
             # 'float96', np.float96), # not supported by numpy on macOS on amd64, apparantly
-            ('float128', np.float128),
+            # ('float128', np.float128),
```

Comment:
Just delete it if it's not gonna be used, but are there some platforms that do use this functionality still?